src/emqx.appup.src

```diff
@@ -6,7 +6,8 @@
      {load_module,emqx_connection,brutal_purge,soft_purge,[]},
      {load_module,emqx_ws_connection,brutal_purge,soft_purge,[]},
      {load_module,emqx_cm,brutal_purge,soft_purge,[]},
-     {load_module,emqx_app,brutal_purge,soft_purge,[]}]},
+     {load_module,emqx_app,brutal_purge,soft_purge,[]},
+     {apply, application, set_env, [gen_rpc, insecure_auth_fallback_allowed, true]}]},
```

Comment:
Not sure if schema is reloaded during hot upgrade, so I added this upgrade instruction just in case.